package.json

```diff
@@ -14,9 +14,13 @@
     "grunt-sass": "^1.0.0",
     "grunt-contrib-less": "1.3.0",
     "grunt-less-to-sass": "0.0.10",
-    "jit-grunt": "^0.9.1"
+    "jit-grunt": "^0.9.1",
+    "swagger-ui": "^2.2.5"
   },
   "devDependencies": {
     "grunt-contrib-watch": "~0.6.1"
+  },
+  "scripts": {
+    "postinstall": "rm -rf public/swagger-ui; cp -r node_modules/swagger-ui/dist public/swagger-ui; sed -i .orig \"s/defaultModelRendering: 'schema'/defaultModelRendering: 'model'/\" public/swagger-ui/index.html; sed -i .orig 's/url = \".*\"/url = \"..\\/..\\/api\\/v1\\?swagger\"/' public/swagger-ui/index.html"
```

Comment:
We talked about this a little in person yesterday, but I think we may not be talking about the same thing. Just in case clarification is needed: I rely on [this task](https://github.com/vufind-org/vufind/blob/master/build.xml#L295) to create VuFind releases (`phing package` from the command line). Since we now have an additional step to get all of the files in place for distributing VuFind, I need to be sure when I run the package task that npm has done its job. It doesn't matter from this perspective whether we use grunt or npm -- we just need to be sure that something puts the files where they belong before we zip them up for distribution.